### PR TITLE
add -f option to run specific test file

### PR DESCRIPTION
**Pull request by jeremyeaton89**
Agnostic approach to running a specific test file. 

former PR: https://github.com/wix/detox/pull/502

### CODE REVIEW

**Comment by jeremyeaton89** [comment]
@Cederman had reviewed the original PR

File: detox/local-cli/detox-test.js [hunk 1]
```diff
@@ -24,11 +24,13 @@ program
...
     'Run platform specific tests. Runs tests with invert grep on \':platform:\', '
     + 'e.g test with substring \':ios:\' in its name will not run when passing \'--platform android\'')
+  .option('-f, --file [path]',
+    'Specify test file to run')
   .parse(process.argv);
 
 const config = require(path.join(process.cwd(), 'package.json')).detox;
 
-const testFolder = getConfigFor('specs', 'e2e');
+const testFolder = getConfigFor('file', 'specs', 'e2e');
```
Comment (rotemmiz) [inline detox/local-cli/detox-test.js]: `getConfigFor` has two params, this puts `file` as key, pushing `specs` as default value, ignoring `e2e` altogether. Am I missing something ?

Comment (rotemmiz) [reply detox/local-cli/detox-test.js]: I can be a good idea to extend `getConfigFor` to support multiple keys

**Comment by jeremyeaton89** [comment]
is the test failure on my end? https://travis-ci.org/wix/detox/jobs/352531838

**Pull request merged**